packages/salesforcedx-vscode-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -27,35 +35,32 @@ export interface LaunchRequestArguments
 
 export class ApexReplayDebug extends DebugSession {
   public static THREAD_ID = 1;
-  protected logFile: LogContext;
+  protected logContext: LogContext;
+  private breakpointUtil: BreakpointUtil;
+  private initializedResponse: DebugProtocol.InitializeResponse;
 
   constructor() {
     super();
     this.setDebuggerLinesStartAt1(true);
     this.setDebuggerPathFormat('uri');
+    this.breakpointUtil = new BreakpointUtil();
   }
 
   public initializeRequest(
     response: DebugProtocol.InitializeResponse,
     args: DebugProtocol.InitializeRequestArguments
   ): void {
-    response.body = {
-      supportsCompletionsRequest: false,
-      supportsConditionalBreakpoints: false,
-      supportsDelayedStackTraceLoading: false,
-      supportsEvaluateForHovers: false,
-      supportsExceptionInfoRequest: false,
-      supportsExceptionOptions: false,
-      supportsFunctionBreakpoints: false,
-      supportsHitConditionalBreakpoints: false,
-      supportsLoadedSourcesRequest: false,
-      supportsRestartFrame: false,
-      supportsSetVariable: false,
-      supportsStepBack: false,
-      supportsStepInTargetsRequest: false
-    };
-    response.success = true;
-    this.sendResponse(response);
+    this.initializedResponse = response;
+    this.sendEvent(new Event(GET_LINE_BREAKPOINT_INFO_EVENT));
```

Comment:
When debugger adapter first starts, send an event to the extension to get line breakpoint info.